experiments/find_uncovered_api.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 The GraphicsFuzz Project Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys, json, subprocess
+
+json_string = subprocess.check_output(
+    ["gcov", "-i", "-t", "libVulkan.cpp.o"],
+    cwd="swiftshader/build-coverage/src/Vulkan/CMakeFiles/vk_swiftshader.dir"
+)
```

Comment:
This code is correct, but for readability I would prefer to clearly define the command and the working dir before calling subprocess.check_output(). Also, use long version of option flags for clarity, and  comment the reason why you need a particular directory:

```
cmd = ["gcov", "--json-format", "--stdout", "libVulkan.cpp.o"]
# Gcov must be called in a precise location to find the gcda file
workdir = "swiftshader/build-coverage/src/Vulkan/CMakeFiles/vk_swiftshader.dir"
json_string = subprocess.check_output(cmd, cwd=workdir)
```